OpenMarket/OpenMarket/Controller/RegisterViewController.swift

```diff
@@ -0,0 +1,170 @@
+//
+//  RegisterViewController.swift
+//  OpenMarket
+//
+//  Created by 우롱차, Donnie on 2022/05/25.
+//
+
+import UIKit
+
+final class RegisterViewController: RegisterEditBaseViewController {
+    
+    private enum Constant {
+        static let navigationTitle = "상품등록"
+    }
+    
+    private let picker = UIImagePickerController()
+    private let productListUseCase = ProductListUseCase()
+    
+    private lazy var addImageButton: UIButton = {
+        let imageButton = UIButton()
+        let image = UIImage(systemName: "plus")
+        imageButton.setImage(image, for: .normal)
+        imageButton.backgroundColor = .systemGray5
+        imageButton.addTarget(self, action: #selector(addImage), for: .touchUpInside)
+        return imageButton
+    }()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        picker.delegate = self
+        setBaseImage()
+        navigationItem.title = Constant.navigationTitle
+    }
+}
+
+// MARK: - Method
+extension RegisterViewController {
+    func addImageToStackView(image: UIImage){
```

Comment:
외부에서 호출하지 않는 메서드는 `private` 붙여주세요 🙏 
추가로 띄어쓰기도 빠진 것 같네요 +_+
```suggestion
    func addImageToStackView(image: UIImage) {
```

Reply:
키워드 추가 및 수정하였습니다.